### PR TITLE
Feature Home page

### PR DESCRIPTION
**Pull request by Avin008**
### Homepage Includes:

ecommerce home

### CODE REVIEW

**Review by kotalwar-krishna — APPROVED** [review]
good to merge

File: index.html [hunk 1]
```diff
@@ -0,0 +1,175 @@
...
+    <h2 class="main-heading">Best Anime Merch</h2>
+
+    <!-- main container -->
+    <main class="main-container">
+      <!-- card -->
+
+      <div class="saiyan-explore-card">
+        <a href="#">
+          <div class="card__img__container">
+            <img class="card__img" src="/Assets/img/img1.webp" alt="card-img" />
```
Comment (Nagashree01) [inline index.html:90]: you can specify better alternate text for image

File: style.css [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+  background-color: purple;
+  padding: 1rem;
+}
+
+.links-container > a {
+  font-weight: 600;
+  border: 1px solid black;
+  padding: 1rem;
+  text-decoration: none;
+  background-color: #404040;
```
Comment (Nagashree01) [inline style.css]: you can use css variables here. Saves your time

**Review by Nagashree01 — APPROVED** [review]
Good to merge

**Pull request merged**